app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
+    <string name="msg_imported">Imported</string>
+    <string name="msg_exported">Exported</string>
+    <string name="msg_export_auth_required">Authenticate before exporting.</string>
+    <string name="msg_export_auth_missing">Turn on screen lock to require authenticating to export servers and passwords.</string>
+    <string name="msg_invalid_key_file">This file is not a valid private key.</string>
     <string name="msg_invalid_vnc_uri">Invalid VNC URI</string>
-    <string name="msg_copied_to_clipboard">Copied!</string>
-    <string name="msg_key_compat_mode_help">Some servers don\'t support direct Unicode input. Compatibility Mode tries to send key events in legacy format.\n\nTry enabling/disabling this if you are having trouble entering some characters.</string>
-    <string name="tip_empty_server_list">Server list is empty.\nUse the \'<b>+</b>\' button to add a Server, or\nUse the top Urlbar to directly connect to a server.</string>
-    <string name="tip_urlbar">You can enter hostname or IP address.\n\'vnc://\' URIs are also supported.</string>
-    <string name="tip_discovery">Discovery can automatically find supported servers\nrunning on your network.</string>
+    <string name="msg_copied_to_clipboard">Copied</string>
+    <string name="msg_key_compat_mode_help">Compatiblity mode can help input characters on servers that don\'t support Unicode.</string>
+    <string name="tip_empty_server_list">Click \'<b>+</b>\' to add a server, or\nuse the top bar to connect to one.</string>
+    <string name="tip_urlbar">Enter a hostname or IP address.\n\'vnc://\' URIs are also supported.</string>
+    <string name="tip_discovery">Discovery can find servers on your network.</string>
 
     <!-- Accessibility descriptions -->
     <string name="desc_navigation_btn">Open Navigation Drawer</string>
     <string name="desc_settings_btn">Open Settings</string>
-    <string name="desc_saved_servers_tab">Saved Servers</string>
-    <string name="desc_discovered_servers_tab">Discovered Servers</string>
-    <string name="desc_add_new_server_btn">Add new Server</string>
-    <string name="desc_save_discovered_server_btn">Save this Server</string>
-    <string name="desc_discovery_btn">Start Server Discovery</string>
-    <string name="desc_discovery_running">Discovering Servers on you network</string>
-    <string name="desc_keyboard_btn">Open Keyboard</string>
-    <string name="desc_virtual_keys_btn">Show Virtual Keys</string>
-    <string name="desc_reset_zoom_btn">Reset Zoom to 100%</string>
-    <string name="desc_retry_connection_btn">Retry Connection</string>
-    <string name="desc_back_btn">Go Back</string>
-    <string name="desc_clear_url_btn">Clear URL</string>
+    <string name="desc_saved_servers_tab">Saved</string>
+    <string name="desc_discovered_servers_tab">Discovered</string>
+    <string name="desc_add_new_server_btn">Add new</string>
+    <string name="desc_save_discovered_server_btn">Save this</string>
```

Comment:
Please don't shorten accessibility description. They should to provide some context about the action.

Reply:
I think those should have different tags, but why not just "Save"?